DEVELOPER.md

```diff
@@ -114,6 +114,17 @@ networks:
     assignments:              # assign static addresses to devices
       magda-core: 1
 
+# not part of magda conf, only to illustrate tunnel usage
+  - vid: 13
+    untagged: true            # don't tag this vlan - useful if the corerouter is plugged into a normal home router
+    name: uplink
+    role: ext
+    tunnel_wan_ip: 192.168.1.2/24   # put here the address of the corerouter inside the uplink network
```

Comment:
this is a network, not only an ip. I suggest to change this to "put the addess of the core router inside the uplink network here together with the netmask of the uplink network.